### PR TITLE
Clarify the L2 voting process

### PR DESCRIPTION
**Pull request by matthewp**

### CODE REVIEW

File: GOVERNANCE.md
Comment (FredKSchott) [inline GOVERNANCE.md:119]: - mention simple vs. overwhelming majority
- actually, everything added here feels relevant to both membership voting, not just committer. Does it make more sense to add this to the existing bullet point as "~core~ contributor membership" and then also remove the point about private messaging for both? I'd be +1 on that change.

**Comment by FredKSchott** [comment]
LGTM! Remember that governance changes require a majority approval before merging + 3 day window.

File: GOVERNANCE.md
Comment (natemoo-re) [inline GOVERNANCE.md:118]:
```suggestion
- Contributor membership (L2 and L3): discussion conducted via a temporary Discord channel open to qualified contributors for up to 3 days. Acceptance requires an overwhelming majority (over 70%) vote conducted by privately messaging a steward. Funneling both assenting and dissenting votes directly through stewards allows for anonymity when discussing the merits of a potential contributor.
```

**Review by natemoo-re — CHANGES_REQUESTED** [review]
Just cleaning up some wording

**Review by natemoo-re — APPROVED** [review]

**Comment by FredKSchott** [comment]
@drwpow waiting for your vote to merge!

**Comment by FredKSchott** [comment]
3 days have elapsed, @drwpow automatically marked as abstaining. 
@natemoo-re @matthewp @FredKSchott marked as approved. 

PR approved!

**Pull request merged**
